examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
Got the modification testing, btw is there an easier way to handle testing on my branch.

Generally, I clone the ingenue/pkg.{name}, then pull the branch I am testing from https://github.com/ruslo/hunter or my own hunter repo. Then push that to a pkg.{name} branch on my repo that starts a travis/appveyor test. Once I get the test to succeed then I have to backout the merge from the test brand and then setup a PR to ingenue. Am I doing something more than needs to be or is that just the way it is?